packages/docs-site/docs/ref/style/usage.md

```diff
@@ -232,19 +234,34 @@ Set A, B
 
 Then, only one of mappings `x -> A; y -> B` and `x -> B; y -> A` triggers the Style block.
 
-### Style Block Body
+### Reserved Variables
+
+Within a _style_ block body, some variable names are reserved for metadata purposes:
 
-The body of a style block lists out computations, shape declarations, and constraint / objective definitions that define how to draw elements of a diagram.
+- `match_count` is an integer that refers to the number of times that this _style_ blocks will be triggered (or matched) in total; and
+- `match_id` is the 1-indexed ordinal of this current matching.
+
+These values can directly be read or overwritten within the style block body, or overwritten, if needed.
```

Comment:
This statement is a bit confusing since if says "overwritten" twice, I'm not quite sure what it means; could you rephrase it to be more clear?

Reply:
Typo - fixed